docs/unified/msi/firmware-update.md

```diff
@@ -14,20 +14,35 @@ Alternatively, it can be checked in the `BIOS Setup Menu`.
 
 ## Prerequisites
 
-Before proceeding, ensure that the firmware protections are disabled in
+Depending on firmware version (1) there may be manual steps required to ensure that
+the firmware can be updated.
+{ .annotate }
+
+1. v0.9.0 for Z790-P and v1.1.2 for Z690-A introduced support for Firmware
+   Update Mode
+
+### Firmware Update Mode
+
+If the currently installed Dasharo version supports Firmware Update Mode, follow
+the steps outlined in
+[generic Firmware Update documentation](../../guides/firmware-update.md#firmware-update-mode).
+
```

Comment:
@mkopec please add similar information to NovaCustom firmware update page: https://docs.dasharo.com/unified/novacustom/firmware-update/.

Reply:
Added